### DISCUSSION TITLE
Kamon health check endpoint

### DISCUSSION THREAD

**Discussion by vpovarna** [kamon-io/Kamon#1140] [Q&A]
Hi,
Does Kamon offers a health check endpoint similar with actuator health endpoint? 
Thanks

**Comment by ivantopo** [comment]
Hey @vpovarna, have you tried using the Kamon Status Page? If you set up Kamon with the Bundle or added the `kamon-status-page` dependency then it should be available at `http://localhost:5266`. It is only meant to show Kamon's components and health for diagnostic purposes, though.

**Comment by vpovarna** [comment]
I saw the status page url. I was looking for something more customisable, where you can add custom services status.
